src/librustc/Cargo.toml

```diff
@@ -11,6 +11,7 @@ crate-type = ["dylib"]
 [dependencies]
 arena = { path = "../libarena" }
 bitflags = "1.0"
+either = "1.5.0"
```

Comment:
I think this is unused now and you can remove it.

Reply:
Ah yeah true